### PR TITLE
10 notify signing error

### PR DESCRIPTION
**Pull request by turadg**
fixes: #10

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ const App = (_props: Props) => {
       setMetricsIndex,
       setGovernedParamsIndex,
       setInstanceIds,
-    });
+    }).catch((err: Error) => console.error('got watchContract err', err));
```
Comment (samsiegart) [inline src/App.tsx:72]: Worth alerting here too? (Maybe if/when we get time we can add a toast instead of alert)

**Review by samsiegart — APPROVED** [review]
One question/suggestion but better than nothing for now so LGTM

File: src/App.tsx (hunk 1 above)
Comment (turadg) [reply src/App.tsx:72]: I figured if this part fails there's not anything they can do. for the signing error they can get more funds.

File: src/utils/displayFunctions.ts [hunk 2]
```diff
@@ -8,6 +8,11 @@ import { IST_ICON } from 'assets/assets';
 import type { BrandInfo } from 'store/app';
 import { wellKnownBrands } from 'config';
 
+export const notifyError = err => {
+  console.log(err);
+  window.alert(err);
```
Comment (dckc) [inline src/utils/displayFunctions.ts:13]: Not alert(err.message)?

Comment (turadg) [reply src/utils/displayFunctions.ts:13]: That would omit "Error: ".
Screen Shot 2022-11-04 at 7 36 09 AM

Comment (dckc) [reply src/utils/displayFunctions.ts:13]: ok. I was worried about `[Object object]` or some such. thanks for the detail.

**Pull request merged**